include/cloudbreak.bash

```diff
@@ -71,13 +71,13 @@ cloudbreak-conf-tags() {
     env-import DOCKER_TAG_FREEIPA 2.55.0-b32
     env-import DOCKER_TAG_ULUWATU 2.55.0-b32
 
-    env-import DOCKER_TAG_IDBMMS 1.0.0-b6009
-    env-import DOCKER_TAG_WORKLOADIAM 1.0.0-b6009
-    env-import DOCKER_TAG_ENVIRONMENTS2_API 1.0.0-b6009
-    env-import DOCKER_TAG_DATALAKE_API 1.0.0-b6009
-    env-import DOCKER_TAG_DISTROX_API 1.0.0-b6009
-    env-import DOCKER_TAG_AUDIT 1.0.0-b6009
-    env-import DOCKER_TAG_DATALAKE_DR 1.0.0-b6009
+    env-import DOCKER_TAG_IDBMMS 1.0.0-b6156
+    env-import DOCKER_TAG_WORKLOADIAM 1.0.0-b6156
+    env-import DOCKER_TAG_ENVIRONMENTS2_API 1.0.0-b6156
+    env-import DOCKER_TAG_DATALAKE_API 1.0.0-b6156
+    env-import DOCKER_TAG_DISTROX_API 1.0.0-b6156
+    env-import DOCKER_TAG_AUDIT 1.0.0-b6156
+    env-import DOCKER_TAG_DATALAKE_DR 1.0.0-b6156
```

Comment:
[1.0.0-b6156](https://release.infra.cloudera.com/manifest_info?stack=THUNDERHEAD&release=1.0.0-b6156) is a failed build on the RE Portal. I think [1.0.0-b6159](https://release.infra.cloudera.com/manifest_info?stack=THUNDERHEAD&release=1.0.0-b6159) is the earliest green one containing your change.

Reply:
Updated